### ISSUE TITLE
Add a time played somewhere. (Current session and total time online)

### ISSUE THREAD

**Issue by Kmtfwtm** [runelite/runelite#12782]
I’ve used RuneLite for a long time now but this is something I’ve really missed from my old proffered client! :( I like to know how long I’ve played Rs in a day.

**Comment by ShaunDreclin** [comment]
`Report Button` plugin covers this already, I think?

**Comment by Kmtfwtm** [comment]
> `Report Button` plugin covers this already, I think?

Yes that has the time played in that sessions but I would like to know my total played time while client has been open.

**Comment by ShaunDreclin** [comment]
Ahh gotcha so like a `Login Timer` option that doesn't reset when you log out. That'd be a useful addition!
